src/data/tokens.json

```diff
@@ -1916,6 +1916,48 @@
         "name": "Axelar Wrapped LAVA",
         "is_native": false,
         "address": "ibc/2C675DF5F8A49C2685775C6A692015A3B059FA6656B4DBB4DF4052F84F7DEDEA"
+      },
+      "cbbtc-satoshi": {
+        "symbol": "axl-cbBTC",
```

Comment:
why is this the only asset going to be added with a dash, while others in this list won't have a dash?